src/language-js/options.js

```diff
@@ -36,6 +36,15 @@ const options = {
     oppositeDescription:
       "Do not print semicolons, except at the beginning of lines which may need them.",
   },
+  experimentalOperatorPosition: {
```

Comment:
The options defined in this file appeared to follow alphabetical order with a few deviations, so I put this next to and alphabetically before `experimentalTernaries`.